components/WelcomeScreen.js

```diff
@@ -0,0 +1,28 @@
+import { Button, Jumbotron } from 'react-bootstrap'
+
+const WelcomeScreen = ({ handleSignup }) => (
```

Comment:
Just a note that for logged-in users refreshing their browser, this screen will flash briefly before the account contents is shown.

Reply:
Yea, I'm seeing as well. I think we should try to address when we swap out use-auth0-hooks for the Auth0 library.